### DISCUSSION TITLE
هلو جولدن

### DISCUSSION THREAD

**Discussion by ikd5** [community/community#118156] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

كيفك

**Comment by bydevmar** [comment]
Hello GOLDEN ARABIC ;

**Comment by LiteBrite82** [comment]
Hi @ikd5 

Thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one. [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md)
